node/cmd/guardiand/node.go

```diff
@@ -225,7 +225,7 @@ func init() {
 
 	alphRPC = NodeCmd.Flags().String("alphRPC", "", "Alephium RPC URL (required)")
 	alphApiKey = NodeCmd.Flags().String("alphApiKey", "", "Alphium RPC api key")
-	alphMinConfirmations = NodeCmd.Flags().Uint8("alphMinConfirmations", 10, "The min confirmations for alephium tx")
+	alphMinConfirmations = NodeCmd.Flags().Uint8("alphMinConfirmations", 1, "The min confirmations for alephium tx")
```

Comment:
1. What is the default value for ETH?
2. Do you have a check somewhere for mainnet on the minimal number of confirmations?

Reply:
The default value for ETH is 1, the purpose of `minimalConfirmations` is that the guardian node will wait `max(minimalConfirmations, consistencyLevel)` blocks before signing the message, `consistencyLevel` is from the contract.